core/src/main/scala/it/cwmp/authentication/AuthenticationService.scala

```diff
@@ -47,9 +48,10 @@ object AuthenticationService {
             HttpHeaderNames.AUTHORIZATION.toString,
             authHeader)
           .sendFuture()
-          .map(res => res statusCode() match {
-            case 201 => res.bodyAsString().get
-            case code => throw new HTTPException(code)
```

Comment:
Queste modifiche credo che rompano i miei test, da controllare quando risolverai i conflitti

Reply:
non sembrano dare errore. Comunque qui vedi come fare la trasform del future in un altro future. Molto utile nei test